src/main/java/com/teenthofabud/wizard/nandifoods/wms/settings/unit/uom/resource/impl/UOMController.java

```diff
@@ -64,43 +62,55 @@ public ResponseEntity<Void> postUOM(@RequestBody @Valid UOMForm form) {
         return ResponseEntity.created(location).build();
     }
 
-    private UOMDto patchUOM(JsonPatch jsonPatch) throws JsonProcessingException, JsonPatchException {
+    private UOMDto patchUOM(JsonPatch jsonPatch) throws UOMException {
```

Comment:
do not modify legacy patch as this will be deprecated in favour of new patch techniques